### PR TITLE
Added SWI-Prolog client

### PR DESCRIPTION
**Pull request by JanWielemaker**
As of the next devel release (8.3.9), SWI-Prolog will ship with a
library(redis).

### CODE REVIEW

File: clients.json [hunk 1]
```diff
@@ -1875,5 +1875,15 @@
...
     "authors": [],
     "active": true
+  },
+
+  {
+    "name": "Redis library for SWI-Prolog",
+    "language": "Prolog",
+    "repository": "https://github.com/SWI-Prolog/packages-redis",
+    "description": "Prolog redis client that exploits SWI-Prolog's extensions such as strings for compact replies and threads to deal with publish/subscribe.",
+    "authors": ["Jan Wielemaker", "Sean Charles"],
```
Comment (itamarhaber) [inline clients.json:1885]: These should be Twitter handles if you use it. If not, just remove the key.

**Review by itamarhaber — COMMENTED** [review]
Hello @JanWielemaker - my deepest apologies for missing this until now. One comment before merging.

File: clients.json (hunk 1 above)
Comment (JanWielemaker) [reply clients.json:1885]: Thanks for picking this up.   I had forgotten about this.   Would still be happy to have an official pointer.   I don't understand your remark.   I'm not into Twitter.  Please handle the PR as you seem fit.  

I should update the README.  The current library is far more mature than what is suggested there.

Comment (itamarhaber) [reply clients.json:1885]:
```suggestion
```

**Review by itamarhaber — APPROVED** [review]

**Comment by zuiderkwast** [comment]
Thanks @JanWielemaker!

**Pull request merged**